util/recreate_all_migrations

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-if [ ! -f tests/docker/tests_single_server/docker-compose.yml ]; then
+if [ ! -f util/.serverpodUtilRoot ]; then
```

Comment:
Suggestion: Name the `.serverpodUtilRoot` file as either snake_case or kebab-case. Camel case is not so common for file names.